psi4/driver/p4util/util.py

```diff
@@ -138,13 +138,12 @@ def set_memory(inputval):
         units = ''
     # Handle memory given as a string
     if isinstance(inputval, basestring):
```

Comment:
@amjames Good catch! We could add a new test in `tests/python/*` which tests things like `set_output_file`, `set_memory`, etc.

@dsirianni In python 3, `isinstance(s, str)` is sufficient which is also backwards compatible.